pkg/terra/utils.go

```diff
@@ -89,10 +97,40 @@ func ContractConfigToOCRConfig(in []byte) ([]byte, error) {
 	if len(in) != 33 {
 		return nil, fmt.Errorf("invalid config length: expected 33 got %d", len(in))
 	}
-	const byteWidth128 = 16
-	padding := make([]byte, 8) // padding to convert int to 192
 	version := in[0:1]
-	min128 := in[1 : byteWidth128+1]
-	max128 := in[1+byteWidth128:]
-	return bytes.Join([][]byte{version, padding, min128, padding, max128}, []byte{}), nil
+	min, err := parse128BitSignedInt(in[1 : byteWidth128+1])
+	if err != nil {
+		return nil, err
+	}
+	max, err := parse128BitSignedInt(in[1+byteWidth128:])
+	if err != nil {
+		return nil, err
+	}
+	minBytes, err := median.ToBytes(min)
+	if err != nil {
+		return nil, err
+	}
+	maxBytes, err := median.ToBytes(max)
+	if err != nil {
+		return nil, err
+	}
+	return bytes.Join([][]byte{version, minBytes, maxBytes}, []byte{}), nil
+}
+
+func parse128BitSignedInt(s []byte) (*big.Int, error) {
```

Comment:
On Solana, I think we use `gagliardetto/binary` [Int128](https://github.com/gagliardetto/binary/blob/master/u128.go#L195) type. Could we also reuse that here?